pom.xml

```diff
@@ -374,6 +385,18 @@
                             <fromFile>transaction-category-collector-client-api-v2.0.0.yaml</fromFile>
                         </configuration>
                     </execution>
+
+                    <execution>
+                        <id>download-pocket-tailor-client-api-spec</id>
+                        <goals>
+                            <goal>download-single</goal>
+                        </goals>
+                        <phase>generate-sources</phase>
+                        <configuration>
+                            <url>https://artifacts.backbase.com/specs/pocket-tailor/</url>
```

Comment:
BB-Fuel should only contain dependencies that are released to repo. In order to point to specs on artifacts, this needs to be marked a pre-release PR.

Reply:
I will indeed mark it with a todo as I already did for the pocket-tailor dependencies.